evm/computation.py

```diff
@@ -133,6 +133,10 @@ def should_return_gas(self):
     def should_erase_return_data(self):
         return self.is_error and self._error.erases_return_data
 
+    @property
+    def memory_bytes(self):
```

Comment:
We've tended to stay away from APIs that are only used in testing.  What do you think about just reaching down into `computation._memory.bytes` in the tests and removing this property?

Reply:
Sure, I wasn't very comfortable to modify all the tests .